src/OTSession.js

```diff
@@ -35,8 +35,7 @@ export default class OTSession extends Component {
     const updateSessionProperty = (key, defaultValue) => {
       if (shouldUpdate(key, defaultValue)) {
         const value = useDefault(this.props[key], defaultValue);
-        const signalData = sanitizeSignalData(value);
-        OT.sendSignal(signalData.signal, signalData.errorHandler);
+        this.sendSignal(value)
```

Comment:
Minor request: add the trailing semicolon.

Reply:
updated!